specification/network/resource-manager/Microsoft.Network/stable/2021-08-01/expressRouteProviderPortLocationList.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Can you move example json to ./examples folder?

Reply:
Done ..Moved to example folder